internal/cephfs/core/volume.go

```diff
@@ -241,24 +233,6 @@ func (s *subVolumeClient) CreateVolume(ctx context.Context) error {
 		return err
 	}
 
-	// create subvolumegroup if not already created for the cluster.
```

Comment:
Apart of documentation update, You might also need below changes/testing

* Upgrade testing to ensure existing PVC works without any problem
* E2E tests to create the subvolumegroup
* Remove extra in-memory check for subvolumegroup

Reply:
> Upgrade testing to ensure existing PVC works without any problem

@Madhu-1 existing PVC are already in a subvolumegroup. So, I guess there won't be any problem.

Reply:
Yes thats correct just need to test and ensure it works fine, This is already part of upgrade testing i believe

Reply:
I have tested it, It works fine